spec.html

```diff
@@ -10360,8 +10360,16 @@ <h1>Static Semantics: UTF16Encoding ( _cp_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-utf16decode" aoid="UTF16Decode">
-      <h1>Static Semantics: UTF16Decode ( _lead_, _trail_ )</h1>
+    <emu-clause id="sec-utf16encode" aoid="UTF16Encode">
+      <h1>Static Semantics: UTF16Encode ( _text_ )</h1>
```

Comment:
I feel somewhat negatively about having both `UTF16Encoding` and `UTF16Encode` (because the names are too similar). Unfortunately I don't have an obviously superior name to suggest. `UTF16EncodeText`, perhaps?